client/my-sites/sharing/buttons/appearance.jsx

```diff
@@ -101,34 +97,31 @@ class SharingButtonsAppearance extends Component {
 	getReblogLikeOptionsElement() {
 		const {
 			isJetpack,
-			isLikesModuleActive,
 			translate
 		} = this.props;
 
-		if ( ( ! isJetpack || isLikesModuleActive ) ) {
-			return (
-				<fieldset className="sharing-buttons__fieldset">
-					<legend className="sharing-buttons__fieldset-heading">
-						{
-							isJetpack
-								? translate( 'Like', { context: 'Sharing options: Header' } )
-								: translate( 'Reblog & Like', { context: 'Sharing options: Header' } )
-						}
-					</legend>
-					{ this.getReblogOptionElement() }
-					<label>
-						<input
-							name="disabled_likes"
-							type="checkbox"
-							checked={ this.isLikeButtonEnabled() }
-							onChange={ this.onReblogsLikesCheckboxClicked }
-							disabled={ ! this.props.initialized }
-						/>
-						<span>{ translate( 'Show like button', { context: 'Sharing options: Checkbox label' } ) }</span>
-					</label>
-				</fieldset>
-			);
-		}
+		return (
+			<fieldset className="sharing-buttons__fieldset">
+				<legend className="sharing-buttons__fieldset-heading">
+					{
+						isJetpack
+							? translate( 'Like', { context: 'Sharing options: Header' } )
```

Comment:
:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `comment`
- `verb: imperative`

 Would it make sense to reuse one of the above?